daemon/indexing/lib/search.js

```diff
@@ -78,10 +78,13 @@ class Listing {
   /**
    * Searches for listings.
    * @param {string} query - The search query.
+   * @param {array} filters - Array of filter objects
+   * @param {integer} itemsPerPage - number of items to display per page
+   * @param {integer} fromPage - what page to return results from
    * @throws Throws an error if the search operation failed.
    * @returns A list of listings (can be empty).
    */
-  static async search(query, filters) {
+  static async search(query, filters, itemsPerPage, offset) {
```

Comment:
as opposed to itermsPerPage, should this be renamed numberOfItems to be consistent with the definitions in the schema ?

Reply:
sure, yeah let me rename this